frontend/discourse/app/components/empty-topic-filter.gjs

```diff
@@ -50,7 +50,10 @@ export default class EmptyTopicFilter extends Component {
   }
 
   get ctaRoute() {
-    if (this.currentUser.new_new_view_enabled && this.args.newListSubset) {
```

Comment:
When we're in a topics/replies sub-tab, `this.args.newListSubset` is always truthy, but when there a no new topics or replies, we want to redirect to the `ctaRoute` since there are no action defined in `ctaLabelWithAction`.